jaclang/plugin/default.py

```
@@ -69,28 +68,26 @@ class JacFeatureDefaults:
 
     @staticmethod
     @hookimpl
-    def context(session: str = "") -> ExecutionContext:
-        """Get the execution context."""
-        ctx = exec_context.get()
-        if ctx is None:
-            ctx = ExecutionContext()
-            exec_context.set(ctx)
-        return ctx
+    def new_context(
```

create_context, get_context, close_context()

For create_context, we should throw an exception if there is already an existing context.